pom.xml

```diff
@@ -52,6 +52,7 @@
         <module>lib/trino-geospatial-toolkit</module>
         <module>lib/trino-hdfs</module>
         <module>lib/trino-hive-formats</module>
+        <module>lib/trino-lance-file</module>
```

Comment:
Please add a documentation `lance.md` in https://github.com/trinodb/trino/tree/master/docs/src/main/sphinx/connector and update https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/connector.md